students/danwoj/Session04/Prototypes/mailroom_dict.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3.6
+
+# This creates a statis donor table that does not retain values after this program runs
+
+donor_fnames = ['Bill', 'Melvin', 'Daphnie', 'Chauncey', 'Frieda']
+donor_lnames = ['Gates', 'Smith', 'Jones', 'Doe', 'Whatever']
+donations = [[200, 500], [2000.00, 3000.50, 3000.00], [1500.00, 500.25], [400.00], [2000.12, 3000, 4000]]
+
+keys = ['f_name', 'l_name', 'don_amt', 'don_num']
+donor_dict = dict.fromkeys(keys, None)
+donor_dict['f_name'] = donor_fnames
+donor_dict['l_name'] = donor_lnames
+donor_dict['don_amt'] = donations
+donor_dict['don_num'] = []
+
+for i in donor_dict['don_amt']:
+	donor_dict['don_num'].append(len(i))
+
+# This is the function that prints off the donor report
+def print_report():
```

Comment:
this is where it would be good to split out the report generation from the printing -- have this runction return a string to print, but actually print it elsewhere.

when you try to test it -- you'll see why!